src/hdtSkinnedMesh/hdtDispatcher.cpp

```
@@ -49,31 +48,30 @@ namespace hdt
 		if (body0->isStaticOrKinematicObject() && body1->isStaticOrKinematicObject())
 			return false;
-		if (body0->checkCollideWith(body1) || body1->checkCollideWith(body0))
-		{
+		if (body0->checkCollideWith(body1) || body1->checkCollideWith(body0)) {
 			auto rb0 = static_cast<SkinnedMeshBone*>(body0->getUserPointer());
 			auto rb1 = static_cast<SkinnedMeshBone*>(body0->getUserPointer());
 
 			return rb0->canCollideWith(rb1) && rb1->canCollideWith(rb0);
-		}
-		else return false;
+		} else
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)